frontend/www/js/omegaup/components/arena/RunDetailsPopup.vue

```diff
@@ -233,6 +239,10 @@ export default class ArenaRunDetailsPopup extends Vue {
     return this.data?.source;
   }
 
+  getShortGuid(guid: string): string {
+    return guid.substring(0, 8);
+  }
```

Comment:
Acá bien se podría utilizar una propiedad computada en lugar de un método:
```suggestion
  get shortGuid(): string {
    return this.data.guid.substring(0, 8);
  }
```
Y al utilizarla ya lo podrías hacer sin necesidad de colocar los paréntesis